com.valvesoftware.Steam.yml

```diff
@@ -227,8 +226,6 @@ modules:
         mkdir -p /app/bin
         mkdir -p /app/lib/i386-linux-gnu
         mkdir -p /app/lib/debug/lib/i386-linux-gnu
-        mkdir -p /app/lib/i386-linux-gnu/GL
-        mkdir -p /app/lib/i386-linux-gnu/dri/intel-vaapi-driver
```

Comment:
These directories are useful in case when the `.GL32` extension is installed while `.Compat.i386` is not; it allows our wrapper and `steam` to run, while otherwise it hard fails early at flatpak level.

Reply:
I see. I removed the commit.